<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white</string>
  </property>
  <widget class="QPushButton" name="home_button">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>540</y>
     <width>61</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>35</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>60</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
color: black;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_friends_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>540</y>
     <width>61</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>35</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>60</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
color: black;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Friends</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_clock_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>61</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>35</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>60</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
color: black;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check_request_button">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>128</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>128</width>
     <height>35</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>60</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
color: black;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>CHECK REQUEST</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Твои дела</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
